sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/DoublePointData.java

```diff
@@ -16,9 +18,40 @@
 @Immutable
 @AutoValue
 public abstract class DoublePointData implements PointData {
+
+  /**
+   * Creates a {@link DoublePointData}.
+   *
+   * @param startEpochNanos The starting time for the period where this point was sampled. Note:
+   *     While start time is optional in OTLP, all SDKs should produce it for all their metrics, so
+   *     it is required here.
+   * @param epochNanos The ending time for the period when this value was sampled.
+   * @param attributes The set of attributes associated with this point.
+   * @param value The value that was sampled.
+   */
   public static DoublePointData create(
       long startEpochNanos, long epochNanos, Attributes attributes, double value) {
-    return new AutoValue_DoublePointData(startEpochNanos, epochNanos, attributes, value);
+    return create(startEpochNanos, epochNanos, attributes, value, Collections.emptyList());
+  }
+
+  /**
+   * Creates a {@link DoublePointData}.
+   *
+   * @param startEpochNanos The starting time for the period where this point was sampled. Note:
+   *     While start time is optional in OTLP, all SDKs should produce it for all their metrics, so
+   *     it is required here.
+   * @param epochNanos The ending time for the period when this value was sampled.
+   * @param attributes The set of attributes associated with this point.
+   * @param value The value that was sampled.
+   * @param exemplars A collection of interesting sampled values from this time period.
+   */
+  public static DoublePointData create(
```

Comment:
Where will this be called and how (i.e.: who will create the collection of exemplars)?

Reply:
That's a separate PR.   

Preview here: https://github.com/jsuereth/opentelemetry-java/blob/wip-metrics-api/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/MetricDataUtils.java#L85

This PR is ONLY the `DataModel -> Exporter` code (I.e. the stable portions of the specification).